hooks/900-cleanup-etc-var.chroot

```diff
@@ -27,8 +27,14 @@ mv etc/pam.d/* usr/lib/pam.d
 # Prefix all includes with /usr/lib/pam.d for now, we have to do this unfortunately
 # due to a bug in the debian patch for @includes.
 # See filed bug: https://bugs.launchpad.net/ubuntu/+source/pam/+bug/2087827
-# XXX: Remove this line once its fixed upstream
+# XXX: Remove these two replacements once its fixed upstream
 find usr/lib/pam.d/ -type f -exec sed -i -e 's/\@include /\@include \/usr\/lib\/pam.d\//g' {} \;
+for file in usr/lib/pam.d/*-l;
```

Comment:
Maybe a comment on what the `-l` implies could be good

Reply:
added!